themes/vue/layout/partials/ecosystem_dropdown.ejs

```diff
@@ -1,37 +1,37 @@
 <li class="nav-dropdown-container ecosystem">
-  <a class="nav-link">Ecosystem</a><span class="arrow"></span>
+  <a class="nav-link">Ecosistema</a><span class="arrow"></span>
   <ul class="nav-dropdown">
-    <li><h4>Help</h4></li>
+    <li><h4>Ayuda</h4></li>
     <li><ul>
-      <li><a href="https://forum.vuejs.org/" class="nav-link" target="_blank">Forum</a></li>
+      <li><a href="https://forum.vuejs.org/" class="nav-link" target="_blank">Foro</a></li>
       <li><a href="https://chat.vuejs.org/" class="nav-link" target="_blank">Chat</a></li>
       <li><a href="https://www.vuemeetups.org/" class="nav-link" target="_blank">Meetups</a></li>
     </ul></li>
-    <li><h4>Tooling</h4></li>
+    <li><h4>Herramientas</h4></li>
     <li>
       <ul>
         <li><a href="https://github.com/vuejs/vue-devtools" class="nav-link" target="_blank">Devtools</a></li>
         <li><a href="https://cli.vuejs.org/" class="nav-link" target="_blank">Vue CLI</a></li>
         <li><a href="https://vue-loader.vuejs.org" class="nav-link" target="_blank">Vue Loader</a></li>
       </ul>
     </li>
-    <li><h4>Core Libraries</h4></li>
+    <li><h4>Librerias Oficiales</h4></li>
```

Comment:
I think it is `Librerías`, isn't it?

```suggestion
    <li><h4>Librerías Oficiales</h4></li>
```

References: 
- https://dle.rae.es/?id=NFiNIHG
- Found also this: "La palabra librerías, con vocal tónica en la segunda 'i', Lleva tilde."

`Razón:`
"Se produce un hiato formado por una vocal débil acentuada (i,u) junto a una vocal fuerte (a,e,o). Este hecho se marca con una tilde en la vocal débil. Aunque es una palabra llana (o grave) acabada en 'n', 's' o vocal y no llevaría tilde según las reglas generales, la formación de hiato por acento en la vocal débil tiene prioridad sobre cualquier regla."

Reply:
Yes, you're right